src/ActionList/Item.tsx

```diff
@@ -457,4 +478,6 @@ export function Item(itemProps: Partial<ItemProps> & {item?: ItemInput}): JSX.El
       </DividedContent>
     </StyledItem>
   )
-}
+}) as PolymorphicForwardRefComponent<'div', ItemProps>
+
+Item.displayName = 'Item'
```

Comment:
Should the display name actually be `ActionList.Item`?
```suggestion
Item.displayName = 'ActionList.Item'
```